### ISSUE TITLE
[ION-1300] Getting this error a lot for my lambda: "it does not seem like sst dev is running"

### ISSUE THREAD

**Issue by jayair** [sst/sst#4125]
https://github.com/sst/sst/issues/3885

<sub>[ION-1300](https://linear.app/sst-dev/issue/ION-1300/getting-this-error-a-lot-for-my-lambda-it-does-not-seem-like-sst-dev)</sub>

**Comment by SamuelLeapifai** [comment]
I managed to fix this. In my case, I had to delete pyproject.toml from my parent directory. This was my setup:
```
root
- pyproject.toml     # this is what I had to delete
- lambdas folder
  - sst.config.ts
  - hello_lambda folder
    - pyproject.toml     # this stays
    - index.py
  - other_lambda folder
    - pyproject.toml
    - index.py
```

**Comment by c4rloscg** [comment]
**Update**: Reverting to 3.2.73 connects my lambdas with my console again.

---

After upgrading SST from version 3.2.73 to 3.3.5, I started encountering the following issue when invoking my Node.js Lambdas using `npx sst dev`: 

```
{"body":"sst dev is not running"}
```

To troubleshoot, I tried deleting all production and development resources in AWS by running `npx sst remove stage...` and then redeployed everything in dev mode. Unfortunately, this didn’t resolve the issue. 

I also deleted the `.sst` folder, cleared `node_modules`, and reinstalled dependencies using `yarn`. Despite these steps, my Lambdas still don’t seem to connect to the dev environment as expected.

For context, I’m running this setup on WSL in Windows.

**Comment by jayair** [comment]
@c4rloscg might be a different issue but we'll take a look.

**Comment by aaronvg** [comment]
I also get this issue following the hello-world example. My sst version is 3.3.10 using `npx sst version`

https://guide.sst.dev/chapters/create-a-hello-world-api.html

**Comment by jayair** [comment]
@aaronvg what OS?

**Comment by aaronvg** [comment]
macOS apple silicon (m1) Ventura.

Can't seem to get it to connect when I open the browser or curl the endpoint still (unsure if this is an sstv3 issue)

**Comment by jayair** [comment]
So when do you get the `sst dev is not running` error?

**Comment by aaronvg** [comment]
<img width="1574" alt="image" src="https://github.com/user-attachments/assets/fa4102f3-e9db-493c-99bd-e9fd48e7db59">

**Comment by aaronvg** [comment]
<img width="581" alt="image" src="https://github.com/user-attachments/assets/50bc2932-d834-44f5-beb1-e0a9fe67040d">
this console url is also blank page:
<img width="1585" alt="image" src="https://github.com/user-attachments/assets/1e12a593-1b64-4ab8-95f0-153343c93f7e">

**Comment by jayair** [comment]
What does your functions tab show?

**Comment by aaronvg** [comment]
<img width="711" alt="image" src="https://github.com/user-attachments/assets/87fdee35-8135-4476-9e2f-5d84fc5ed9dd">

**Comment by jayair** [comment]
@thdxr

**Comment by thdxr** [comment]
can you share what region your app is in

**Comment by aaronvg** [comment]
us-east-1. I'm gonna try 3.2.73

**Comment by aaronvg** [comment]
alrighty, I manually changed all package.jsons on the hello-world to 3.2.73, ran it and it worked.

I changed it back once more to 3.3.10 and it didn't work again. will stay on 3.2.73.

Unsure how many this affects but I couldn't get past the first tutorial with this problem.

**Comment by thdxr** [comment]
potential fix in v3.3.11

**Comment by c4rloscg** [comment]
> potential fix in v3.3.11

This fix worked for me, thanks! Using 3.3.13 now on WSL in Windows without issues.
Related [comment](https://github.com/sst/sst/issues/4125#issuecomment-2462567851).

**Comment by jayair** [comment]
Great. Closing this for now.
